### PR TITLE
Added respec defines to improve github links

### PR DESCRIPTION
**Pull request by halindrome**
Added issues and version control links.
Removed useExperimentalStyles as it is no longer required.

### CODE REVIEW

File: specs/architecture.html [hunk 1]
```diff
@@ -32,6 +31,18 @@
           wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/83744/status",
 
           issueBase:    "https://github.com/w3c/browser-payment-api/issues/",
+          // additional defines to improve github integration
+          githubAPI:  "https://api.github.com/repos/w3c/webpayments-ig",
```
Comment (zkoch) [inline specs/architecture.html]: Should this point to the IG's Github API?

Comment (halindrome) [inline specs/architecture.html]: Hmm.  Maybe not.  My bad!

**Comment by adrianhopebailie** [comment]
LGTM

**Comment by adrianba** [comment]
Merged as 6def762e2785568881638c34102bfe2f5f64325b

**Pull request closed without merging**